docs/services/gcp-services-guide.md

```diff
@@ -0,0 +1,536 @@
+# Deploying Wordpress to GKE
```

Comment:
Structural comments:

* I would suggest that we add a table of contents to this doc, as it does have some length on it.
* I would suggest that we add a Conclusion section and Next Steps section. Conclusion would summarize what we did, and Next Steps would suggest other documents to read, or other exercises to try. See the [stack guide](https://github.com/crossplaneio/crossplane/pull/803/files#diff-378fbcada1ba53bb6c4e2afb09c18c3aR228) for some examples